data/events/scripts/player.lua

```diff
@@ -93,6 +93,18 @@ function Player:onMoveCreature(creature, fromPosition, toPosition)
 	return true
 end
 
+function Player:onWrapItem(item)
+	local pos = item:getPosition()
+	local house = Tile(pos):getHouse()
+	
+	if house then
```

Comment:
It is correct, but I would rather see this inverted so that return true is at the bottom.